ee-galleon-pack/src/main/resources/layers/standalone/web-console/layer-spec.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" ?>
+<layer-spec xmlns="urn:jboss:galleon:layer-spec:1.0" name="web-console">
+    <dependencies>
+        <layer name="secure-management"/>
+        <layer name="core-management"/>
```

Comment:
Why this?

Reply:
I'm fine with keeping this but a HAL issue to allow it to gracefully live without this would be good.

Reply:
https://issues.redhat.com/browse/HAL-1707